libreoffice-core/sw/source/core/doc/DocumentRedlineManager.cxx

```diff
@@ -1274,6 +1274,13 @@ DocumentRedlineManager::AppendRedline(SwRangeRedline* pNewRedl, bool const bCall
         return AppendResult::IGNORED;
     }
 
+    // MACRO-1266 - Do not create a delete redline for a comment
+    if(pNewRedl->IsAnnotation()) {
+        delete pNewRedl;
+        pNewRedl = nullptr;
```

Comment:
It's pretty much never a good idea to manually delete a class.
Assigning pNewRdl to nullptr is sufficient.

Reply:
Didn't see that, in that case, do what they do! Definitely makes me curious why, but better go with the flow than against it haha